terraform/providers/google/go.mod

```diff
@@ -1,41 +1,41 @@
 module github.com/openshift/installer/terraform/providers/google
 
-go 1.18
+go 1.19
 
-require github.com/hashicorp/terraform-provider-google v1.20.1-0.20230327171628-0dc3bde12208 // v4.59.0
+require github.com/hashicorp/terraform-provider-google v1.20.1-0.20230718215755-3edc574a3a8f // v4.74.0
 
 require (
-	bitbucket.org/creachadair/stringset v0.0.11 // indirect
-	cloud.google.com/go v0.110.0 // indirect
-	cloud.google.com/go/bigtable v1.17.0 // indirect
-	cloud.google.com/go/compute v1.18.0 // indirect
+	bitbucket.org/creachadair/stringset v0.0.8 // indirect
```

Comment:
So even with the `replace` this is being downgraded?

Reply:
v4.74.0 of terraform-provider-google is using v0.0.8, but we are replacing with v0.0.11 and go.sum has entry for v0.0.11. Please correct me, this is as expected isn't it?